Configuration/webapp/app/components/form/Form.tsx

```diff
@@ -64,21 +65,22 @@ export const Form: FC<FormProps> = ({ sectionTitle, items, itemsRestrictions })
   );
 
   return (
-    <Accordion defaultExpanded>
-      <AccordionHeader
-        title={sectionTitle}
-        viewForm={viewForm}
-        viewFormToggle={() => setViewForm((v) => !v)}
-      />
-      <AccordionDetails>
-        {viewForm ? (
+    <>
+      <Accordion defaultExpanded>
+        <AccordionHeader title={sectionTitle} viewFormToggle={() => setIsRawModalOpen(true)} />
+        <AccordionDetails>
           <Stack spacing={2}>
             {Object.entries(itemsRestrictions).map(([key, value]) => renderItem(key, value))}
           </Stack>
-        ) : (
-          <Typography component="pre">{JSON.stringify(items, null, 2)}</Typography>
-        )}
-      </AccordionDetails>
-    </Accordion>
+        </AccordionDetails>
+      </Accordion>
+
+      <RawViewModal
+        open={isRawModalOpen}
+        onClose={() => setIsRawModalOpen(false)}
+        title={`${sectionTitle}`}
```

Comment:
I believe you can skip the template literal here as below:

```suggestion
        title={sectionTitle}
```